src/components/FullJob/index.js

```diff
@@ -33,7 +33,7 @@ const FullJob = ({job, currentUser, isAdmin, approveJob, unPublishJob}) => {
                 {(job.approved === false && job.creator_id === currentUser.uid) || isAdmin === true?
                     <Link
                         to={{
-                            pathname: ROUTES.EDIT_JOB
+                            pathname: `/my/jobs/${job.id}/edit`
```

Comment:
I'm wondering should the single job page URL and this URL read `/my/job/${job.id}/edit` instead?

jobs -> job as it's a single page, also give the main URL will read jobs/codebar.io